docker/service/src/main/java/uk/nhs/adaptors/scr/mappings/from/fhir/ObservationMapper.java

```diff
@@ -11,9 +11,7 @@
 import org.hl7.fhir.r4.model.Period;
 import uk.nhs.adaptors.scr.exceptions.FhirValidationException;
 import uk.nhs.adaptors.scr.models.GpSummary;
-import uk.nhs.adaptors.scr.models.xml.Finding;
-import uk.nhs.adaptors.scr.models.xml.RiskToPatient;
-import uk.nhs.adaptors.scr.models.xml.SocialOrPersonalCircumstance;
+import uk.nhs.adaptors.scr.models.xml.*;
```

Comment:
this will show up in gradle errors - i sent a message on #scr-api in slack for how to stop this automatically happening